core/src/main/scala/kafka/network/SocketServer.scala

```diff
@@ -730,7 +736,8 @@ private[kafka] class Processor(val id: Int,
     }
   }
 
-  private def updateRequestMetrics(response: RequestChannel.Response) {
+  private def updateRequestMetrics(response: RequestChannel.Response): Unit = {
+    // Record the amount of time this request spent on the network thread
```

Comment:
Was this comment added intentionally? The request metrics include more than how much time was spent on the network thread.